src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,34 @@
 package mate.academy;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import mate.academy.lib.Injector;
+import mate.academy.model.Movie;
+import mate.academy.service.MovieService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static final MovieService movieService
```

Comment:
```suggestion
    private final MovieService movieService
```
сервіси краще не робити статичними, статичні змінні блочать полфіморфізм і ускладнюють тестування